_docs_integrate/migration-from-v6-to-v7.md

```diff
@@ -0,0 +1,73 @@
+---
+# Start automatic generation
+permalink: integrate/migration-from-v6-to-v7
+published: true
+title: "Migration From v6 to v7"
+type: scenario
+toc: true
+properties:
+  - id: SC123
+  - category: Migration Guides
+  - description: Changes due to release/v7
+  - customer:
+  - component: integrate
+  - level:
+  - implementation status: DOCS ONLY
+  - documentation status: DONE
+  - published: true
+  - link: migration-from-v6-to-v7
+require:
+required_by:
+# End automatic generation
+---
+
+The [Runtime](https://github.com/nmshd/runtime) of enmeshed has recently been updated from version 6 to version 7.
+Accordingly, a new version of the [Connector](https://github.com/nmshd/connector) has also been released to make the updated Runtime available to Integrators of Connectors.
+The version update has resulted in some breaking changes.
+To support the migration of existing systems to the new version, the breaking changes made are listed and explained in this migration guide.
+
+## Step-by-Step Instructions
+
+The step-by-step instructions can be consulted to start the migration to version 7 as an Integrator of a Connector directly.
+[Runtime-specific breaking changes](#runtime-specific-breaking-changes) that do not need to be taken into account when updating the Connector from version 6 to version 7 can be found below.
+
+### Connector Setup
+
+- The data from the database that was used by the Connector of the former version is outdated. This is because the `title` of most [RequestItems]({% link _docs_integrate/data-model-overview.md %}#requestitems) has been removed, for example. For this reason, the old data must be deleted. Alternatively, the database can be deleted as a whole and [set up again]({% link _docs_operate/setup-with-docker-compose.md %}).
```

Comment:
hmmm, I think removed property would be handled quite well from ts-serval, we should maybe remove this explanation..
```suggestion
- The data that was written by pre v7 connectors to the database is not compatible with the Connector v7 version. For this reason, the old data must be deleted. Alternatively, the database can be deleted as a whole and [set up again]({% link _docs_operate/setup-with-docker-compose.md %}).
```

Reply:
I also prefer the wording `the data is not compatible` over the wording `the data is invalid`